[DOCS-8106] Add Cloud SIEM data security doc (#30536)
DataDog/documentation: content/en/data_security/cloud_siem.md

---
title: Cloud SIEM Data Security
disable_toc: false

> aliases:
> - /path-to-old-doc/

Reviewer: ```suggestion
```

further_reading:
- link: "/data_security/"
  tag: "Documentation"
  text: "Review the main categories of data submitted to Datadog"
- link: "/data_security/pci_compliance/"
  tag: "Documentation"
  text: "Set up a PCI-compliant Datadog organization"
---

<div class="alert alert-info">This page is about the security of data sent to Datadog. If you're looking for cloud and application security products and features, see the <a href="/security/" target="_blank">Security</a> section.</div>

## Overview

> A security signal is generated when at least one case defined in a detection rule is matched over a given period of time. You can customize detection rules to provide notification messages that contain specific information about the signal (for example, user ID, IP addresses, and so on) and the triggering group-by values of the signal. Security rules can also use webhooks to send notifications to third-party services. Since data sent to Datadog may contain sensitive information, this document goes over those notification features and what to do if you do not want users to have access to these features.

Reviewer: ```suggestion
Datadog generates a security signal when at least one case defined in a detection rule is matched over a given period of time. You can customize detection rules to provide notification messages that contain specific information about the signal (for example, user ID, IP addresses, and so on) and the triggering group-by values of the signal. Security rules can also use webhooks to send notifications to third-party services.

Because data sent to Datadog may contain sensitive information, this document goes over those notification features and what to do if you do not want users to have access to these features.
```
Can we clarify what you mean by "users"? I can imagine users seeing this and being unsure of whether we mean end users or Datadog users within their organization.

Author: Thanks! I edited to "your users".

## Security rules can use message template variables

When you create a detection rule you can customize the notification message with [notification variables][1], which adds specific information related to the signal. For example, if the following JSON object is associated with a security signal:

```
{
  "network": {
    "client": {
      "ip": "1.2.3.4"
    }
  },

>   "usr": {

Reviewer: Just want to double check whether this should be `user`?

Author: ```suggestion
  "user": {
```

    "id": "user@domain.com"
  },
  "used_mfa": "false"
}
```

> Using `{{@network.client.ip}}` in the notification message displays the IP addresses associated with the signal.

Reviewer: ```suggestion
Using `{{@network.client.ip}}` in the notification message would display the IP address associated with the signal.
```
Small edit to make the example continue, but disregard if this impacts the accuracy!

> Contact [support][2] if you do not want users to be able to add template variables to notification messages.

Reviewer: ```suggestion
Contact [support][2] if you want to prevent users from adding template variables to notification messages.
```

## Security rules can include triggering group-by values in the notification title

> In the **Describe your playbook** section for [log detection rules][3] and [signal correlation rules][4], you can add group-by values in the notification title. For example, if you are grouping by `service`, the service name shows in the title. Uncheck **Include triggering group-by values in notification title** to opt out of adding triggering group-by values to the title.

Reviewer: ```suggestion
In the **Describe your playbook** sections for [log detection rules][3] and [signal correlation rules][4], you can add group-by values in the notification title. For example, if you are grouping by `service`, the service name shows in the title. Uncheck **Include triggering group-by values in notification title** to prevent group-by values from appearing in the title.
```

> Contact [support][2] If you do not want to have the **Include triggering group-by values in notification title** option available.

Reviewer: ```suggestion
Contact [support][2] if you want to remove the **Include triggering group-by values in notification title** option.
```

## Security rules can use webhooks

> <div class="alert alert-warning">If your organization had HIPAA enabled in 2024 or earlier, you may have to reach out to <a href = "https://docs.datadoghq.com/help/">Datadog support</a> to enable webhooks for security rules.</a></div>

Author: ```suggestion
<div class="alert alert-warning">If your organization had HIPAA enabled in 2024 or earlier, reach out to <a href = "https://docs.datadoghq.com/help/">Datadog support</a> to enable webhooks for security rules.</a></div>
```

> Security notifications can be sent to [integrations][5], such as Jira, PagerDuty, and [webhooks][6]. Contact [support][2] if you do not want users to be able to send notifications to a third-party service using webhooks.

Reviewer: ```suggestion
Security notifications can be sent to [integrations][5], such as Jira, PagerDuty, and [webhooks][6]. Contact [support][2] to prevent users from sending notifications to third-party services using webhooks.
```

> 

Reviewer: ```suggestion

## Further reading
{{< partial name="whats-next/whats-next.html" >}}

```

[1]: /security/notifications/variables/?tab=cloudsiem#template-variables
[2]: /help/

> [3]: /security/cloud_siem/log_detection_rules/?tab=threshold#say-whats-happening
> [4]: /security/cloud_siem/signal_correlation_rules#say-whats-happening

Reviewer: ```suggestion
[3]: /security/cloud_siem/log_detection_rules/?tab=threshold#describe-your-playbook
[4]: /security/cloud_siem/signal_correlation_rules#describe-your-playbook
```
